src/main/java/mate/academy/spring/controller/AuthenticationController.java

```diff
@@ -0,0 +1,23 @@
+package mate.academy.spring.controller;
+
+import mate.academy.spring.model.dto.request.UserRequestDto;
+import mate.academy.spring.security.AuthenticationService;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService authenticationService;
+
+    public AuthenticationController(AuthenticationService authenticationService) {
+        this.authenticationService = authenticationService;
+    }
+
+    @PostMapping("/register")
+    public String register(@RequestBody UserRequestDto userRequestDto) {
+        authenticationService.register(userRequestDto.getEmail(),
+                        userRequestDto.getPassword());
+        return "User was registered successfully.";
```

Comment:
I think we don't need this("User was registered successfully.") here. You can read about `ResponseEntity` if you want to return smth